kivy/uix/behaviors/focus.py

```diff
@@ -489,6 +489,12 @@ def _get_focus_next(self, focus_dir):
             else:
                 return None
 
+    def go_left(self,*args):
```

Comment:
Yes, please read https://kivy.org/docs/contribute.html. Also, the function doesn't return anything.